selfdrive/car/honda/hondacan.py

```diff
@@ -6,7 +6,9 @@ def get_pt_bus(car_fingerprint, has_relay):
   return 1 if car_fingerprint in HONDA_BOSCH and has_relay else 0
 
 
-def get_lkas_cmd_bus(car_fingerprint, has_relay):
+def get_lkas_cmd_bus(car_fingerprint, has_relay, openpilot_longitudinal_control=False):
```

Comment:
Maybe rename argument to `radar_present` since that is was actually matters right? Since it forwards the commands?

Reply:
Also add some comments about what all the buses are and why they are different for all the different checks in here. I always forget, so this might be a good place to put an explanation.

Reply:
How about `radar_disabled`?
Should I change all functions I added the parameter `openpilot_longitudinal_control` to be `radar_disabled`?